minutes/2021-05-11.md

```diff
@@ -0,0 +1,43 @@
+# Parsec Community Meeting (11/05/2021) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Paul Howard
+- Marc Meunier
+- Ionut Mihalcea
+- Hugues de Valon
+
+## Agenda
+
+1. Should we add fuzz-testing as a pre-release step?
+2. Communicate the new release process
+3. OnDiskKeyInfoManager's stability for non-breaking changes
+4. Is ECC support for PKCS11 provider required for 0.8.0 release? Could/should we do a 0.7.x in the
+   mean-time, just with that?
+
+## Minutes
+
+- Fuzz testing as part of the Linux Fondation badge requirement. We do not have automated fuzz
+   testing. Should we manually run it duing the code-freeze period? Should we take some time
+   updating? 3-4 weeks to run it.
+- GH issue to track updating of fuzz testing framework
+   ([here](https://github.com/parallaxsecond/parsec/issues/422)). If an error found is important,
+   maybe better to find it before stabilization, rather than during stabilization period rush.
+- Could use the [CNCF Cluster](https://github.com/cncf/cluster) to run it but on which version of
+   Parsec? Continuously?
+- A new release process is available
+   [here](https://parallaxsecond.github.io/parsec-book/contributing/release_process.html). How do we
+   make sure people know about it? Could look at the issue board to track the milestone every week.
+- Milestone 0.8.0 to be put on all relevent issues for an easy view. Attestation and stability are
+   the current main topics.
+- ECC support should be added if we have the availability for the 0.8.0 release. A lot of parts are
+   already there to make it easier. [The issue](https://github.com/parallaxsecond/parsec/issues/421)
+   has been tagged.
+- In the long term: has to change KIM but also manage existing data in the old KIM. Self-contained
+   tool is also possible when Parsec is shutdown.
```

Comment:
I understand what this means but I don't think someone else would - maybe a few more words about what the tool would do?